apps/location/models.py

```diff
@@ -1 +1,45 @@
 # Create your models here.
+from django.db import models
+from django.contrib.auth.models import User
+
+
+
+class Address(models.Model):
+    DISTRICT_CHOICE = (
+        ('Шевченківський'),
+        ('Галицький'),
+        ('Франківський'),
+        ('Залізничний'),
+        ('Сихівський'),
+        ('Личаківський'),
+        )
+
+    country = models.CharField(
+        unique= True,
+        max_length=255,
+        blank=False,
+        verbose_name='Країна'
+    )
+    city = models.CharField(
+        unique= True,
+        max_length=122,
+        blank=False,
+        verbose_name='Місто'
+    )
+    Address = models.OneToOneField(to=country)  # how to do it better?
```

Comment:
only lower case there
You should create FR (https://docs.djangoproject.com/en/2.0/ref/models/fields/#foreignkey)